src/session.ts

```diff
@@ -3,10 +3,12 @@ import { IncomingMessage, ServerResponse } from "http";
 import { nanoid } from "nanoid";
 import MemoryStore from "./memory-store";
 import { isDestroyed, isNew, isTouched } from "./symbol";
-import { Options, Session } from "./types";
+import { Options, Session, SessionRecord } from './types'
```

Comment:
Could you respect the current formatting style?

Reply:
Sorry, I fixed.